src/test/java/org/openstreetmap/atlas/checks/validation/intersections/BigNodeBadDataCheckTest.java

```diff
@@ -73,6 +73,11 @@ private boolean runTest(final Atlas atlas, final Configuration config,
 
         bigNodes.forEach(bigNode ->
         {
+            if (bigNode.nodes().stream().mapToLong(node -> node.getOsmIdentifier())
+                    .anyMatch(number -> number == 812034701L))
+            {
+                logger.info("found something");
+            }
```

Comment:
I'm guessing this was to test something?

Reply:
Yes. I think I was having issues with a test failing somewhere when I was updating to newer `atlas`/`atlas-generator` versions.